charts/weave-policy-agent/values.yaml

```diff
@@ -21,6 +21,10 @@ policy-agent:
     audit:
       # Enable audit functionality
       enabled: false
+      sinks:
+        # Enable writing violations as K8s events
+        k8sEventsSink:
```

Comment:
`k8sEventsSink` sink is not the best sink for the audit mode, because in the audit mode the agent scans all the entities of the cluster so expected to generate a lot of kubernetes events and that could effect the cluster event queue or the visibility of other events

so i think it is better to remove this sink from the default configuration

Reply:
@MostafaMegahid what do u think ?

Reply:
@ahsayde okay I commented it out, but I think it should still be there because I had no idea it existed without it